packages/manager/modules/vps/src/import/helper.service.js

```diff
@@ -0,0 +1,20 @@
+export default class VpsHelperService {
```

Comment:
This should be a util not a service. You don't need to create a object to call these util methods. AngularJs will create a instance of service when used.

Reply:
These functions are dependent on each other so i went with service and not `utils`.